kuma/api/v1/tests/test_views.py

```diff
@@ -330,3 +330,42 @@ def test_search_basic(self):
         assert data['documents']
         assert data['count'] == 5
         assert data['locale'] == 'fr'
+
+
+@pytest.mark.parametrize(
+    'http_method', ['put', 'post', 'delete', 'options', 'head'])
```

Comment:
This is pretty overkill. I don't see a huge benefit in testing that the `require_GET` decorator works. If it's really important that you can't use anything other than `GET` we should probably write and explain the reasoning of that where we use the `require_GET` decorator in views.py

Reply:
My thinking here is again to test the "contract" of the endpoint, so that if that contract changes, the tests will fail. I think `HEAD` makes sense here though, so I'm going to allow that.